src/Sidekick.Business/Apis/Poe/Parser/ParserService.cs

```diff
@@ -33,16 +33,15 @@ public class ParserService : IOnAfterInit, IParserService
             this.statsDataService = statsDataService;
             this.itemDataService = itemDataService;
             this.patterns = patterns;
+            itemNameTokenizer = new ItemNameTokenizer();
         }
 
-        private Regex NewlinePattern;
-        private Regex SeparatorPattern;
+        private const string NEWLINE_PATTERN = "\r\n";
+        private const string SEPARATOR_PATTERN = "--------";
 
         public Task OnAfterInit()
         {
-            NewlinePattern = new Regex("[\\r\\n]+");
-            SeparatorPattern = new Regex("--------");
-
+            // Not needed?
             return Task.CompletedTask;
```

Comment:
If the only line is not Task.CompletedTask, this method (and interface) is to be removed.